tests/cli/cli_test.go

```diff
@@ -127,6 +127,14 @@ func writeNetrc(token, uri string) (*os.File, error) {
 	return f, nil
 }
 
+func deleteNetrc() error {
```

Comment:
Now that I'm looking at this, I'm actually kinda confused why a `.netrc` ever gets written to this directory, since we write the `.netrc` to a tempfile: https://github.com/remind101/empire/blob/remove-netrc/tests/cli/cli_test.go#L109-L128

Maybe it's the login tests?